portfolio/src/main/webapp/index.html

```diff
@@ -22,7 +22,7 @@
   <body data-spy="scroll" data-target="#navbar" data-offset="100">
     <!-- Navigation Bar -->
     <nav id="navbar" class="navbar sticky-top">
-      <a id="name" class="navbar-brand" href="#">Omar Montiel</a>
+      <a id="myName" class="navbar-brand" href="#">Omar Montiel</a>
```

Comment:
To avoid any confusion later on, it might be more straightforward to name this "navbar-name" or something more specific! If another engineer were to join this project, I could see them getting confused by the difference between "name" and "myName".

Reply:
Done!